src/main/java/cart/application/CartItemService.java

```diff
@@ -26,8 +25,12 @@ public List<CartItemResponse> findByMember(Member member) {
         List<CartItem> cartItems = cartItemDao.findByMemberId(member.getId());
         return cartItems.stream().map(CartItemResponse::of).collect(Collectors.toList());
```

Comment:
```suggestion
        return cartItems.stream()
        .map(CartItemResponse::of)
        .collect(Collectors.toList());
```
이런식으로 한줄의 점 하나씩 작성했다면 더 보기 좋았을거 같아요~

Reply:
변경하도록 하겠습니다!